themes/godotengine/layouts/download.htm

```diff
@@ -153,6 +153,11 @@ <h4>Requirements</h4>
           <h4>Instructions</h4>
           <ul>
             <li>Extract and run. Godot is self-contained and does not require installation.</li>
+            <li>
+              If you run into an issue, check the
+              <a href="https://docs.godotengine.org/en/latest/about/troubleshooting.html">Troubleshooting</a>
```

Comment:
This should likely link to `stable` (but first I need to merge `master` in `stable` for the docs).

Reply:
https://docs.godotengine.org/en/stable/about/troubleshooting.html should be valid in half an hour or so, so you can use that. (I synced the docs 2 days ago but `stable` failed building for a non-deterministic reason while `3.2` succeeded with the same code.)